wordle-web/components/keyboard.vue

```diff
@@ -74,8 +87,22 @@ export default class KeyBoard extends Vue {
     if (this.wordleGame.wrongChars.includes(char)) {
       return Letter.getColorCode(LetterStatus.Wrong)
     }
-
     return Letter.getColorCode(LetterStatus.Unknown)
+  }  
+  created(){
+
   }
+   validWords(){
+       const word: Word = this.wordleGame.currentWord;
+       if(word.length === 5){
+           let stringWord: string="";
+           for(let i=0;i<5;i++){
+               stringWord += word.letters[i].char;
+           }
```

Comment:
```suggestion
           let stringWord: string= word.letters.map(x => x.char).join('');
```
Another way you could go about converting this array to a string.